src/interpreter/primitive-props.ts

```diff
@@ -260,6 +260,24 @@ const PRIMITIVE_PROPS: {
 			target.value = await mergeSort(target.value, comp);
 			return target;
 		}),
+		
+		fill: (target: VArr): VFn => FN_NATIVE(async ([val, st, ed], opts) => {
+			const value = val ?? NULL;
+			const start = st && (assertNumber(st), st.value);
+			const end = ed && (assertNumber(ed), ed.value);
+			return target.value.fill(value, start, end), target;
```

Comment:
returnの中でfillせずに分けたい
```suggestion
			target.value.fill(value, start, end);
			return target;
```